### DISCUSSION TITLE
How does this compare to obsidian.nvim?

### DISCUSSION THREAD

**Discussion by einarpersson** [Feel-ix-343/markdown-oxide#195] [General]
Hi!

I currently use the following in tandem
- [obsidian.nvim](https://github.com/epwalsh/obsidian.nvim)
- [render-markdown.nvim](https://github.com/MeanderingProgrammer/render-markdown.nvim)
- [link.vim](https://github.com/qadzek/link.vim)

I like the idea of separating the UI from the underlying logic. I am very happy with `render-markdown.nvim` doing the rendering, and I have disabled rendering for `obsidian.nvim`.

With all that said, I guess my question is in the title. How does this thing (markdown-oxide) compare to `obsidian.nvim`? I understand that the scope is broader / different (I like the idea of leveraging the LSP). But personally I ask, can I replace obsidian.nvim with this? The website says that it is highly compatible with obsidian but I'm not sure what it means. Does it provide the basic functionalities as well?

Eg.
- Browsing my notes in my vault
- Creating new notes inline with links somehow
- Jumping between notes, following links
- Templating?? (eg. recurring frontmatter or specific templates for different notes)
- Hooks?? (eg. on new note creation)

I see some features listed as available and others not yet (eg links to headings). But I guess my general question is: If I want to manage my obsidian vault in neovim, could I do that with just this plugin and not obsidian.nvim (in tandem with rendering plugin)? And does using this plugin have some extra benefit that you know of, right off the bat?

Thanks!

**Comment by Feel-ix-343** [comment]
Hey! We support all of your points except for templating and hook. 

I'd recommend using both obsidian.nvim and oxide and doing so by turning off obsidian.nvim completions. Obsidian.nvim is really good for syncing Obsidian and neovim and supports other obsidian specific features like templating and hooks

**Comment by rbhanot4739** [comment]
Most of the features like creating notes from links, moving b/w notes in addition to browning the vault are supported by obsidian.nvim, so is this plugin only for better completions powered by lsp ?
